src/components/desktop/SidePaneSwitch.js

```diff
@@ -0,0 +1,17 @@
+import { Route, Switch } from 'react-router-dom'
+
+import EntryPane from './EntryPane'
+import MainPane from './MainPane'
+
+const SidePaneSwitch = () => (
+  <Switch>
+    <Route path="/entry/:id">
```

Comment:
Can we add the `exact` property here for clarity?

Reply:
Yes, good idea.